### PR TITLE
Refactor Tag usage

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR refactors the GcnEvent tags, which are used in both the RecentGcnEvents widget and  GcnEvent page, so this uses the same code both places.

### CODE REVIEW

**Review by guynir42 — APPROVED** [review]
LGTM

File: static/js/components/GcnEventPage.jsx [hunk 1]
```diff
@@ -39,6 +39,8 @@ import GcnSelectionForm from "./GcnSelectionForm";
 import ObservationPlanRequestForm from "./ObservationPlanRequestForm";
 import ObservationPlanRequestLists from "./ObservationPlanRequestLists";
 
+import RenderTags from "./RenderTags";
```
Comment (stefanv) [inline static/js/components/GcnEventPage.jsx:42]: Since this is specific to Gcn, consider including Gcn in the component name e.g. `GcnTags`.

**Review by stefanv — COMMENTED** [review]
Suggested a naming change, otherwise looks all good.

File: static/js/components/GcnEventPage.jsx (hunk 1 above)
Comment (mcoughlin) [reply static/js/components/GcnEventPage.jsx]: Good call! Fixed!

**Comment by mcoughlin** [comment]
@stefanv please merge if happy?

**Pull request merged**